dapp_stats/statistics/schemas.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+from pydantic import BaseModel
+from typing import Dict
+
+from dapp_stats.statistics.enums import NodeState
+
+
+class StateLogEntry(BaseModel):
```

Comment:
will this model accept additional keys? it would be good for it to be tolerant enough so it doesn't break if we decide to supplement the structure with additional information in the future... but that can be done with a separate pull request, I think

Reply:
Yes it will ignore additional keys, it only requires defined keys to be there